assets/app/views/_settings-general-info.html

```diff
@@ -0,0 +1,92 @@
+<h3>General information</h3>
+
+
+<div column ng-if="!show.editing">
+  <div row>
+    <div flex><strong>Name:</strong></div>
+    <div flex grow="4" class="project-name">{{projectName}}</div>
+  </div>
+  <div row>
+    <div flex><strong>Display Name:</strong></div>
+    <div row flex grow="4">
+      <div
+        class="project-display-name"
+        ng-if="project | displayName: true">
+        {{project | displayName: true}}
+      </div>
+      <div
+        class="project-display-name no-display-name"
+        ng-if="!(project | displayName: true)">
+        <em>No display name</em>
+      </div>
+    </div>
+  </div>
+  <div row>
+    <div flex><strong>Description:</strong></div>
+    <div row flex grow="4">
+      <div
+        ng-if="project | description">
+        <truncate-long-text
+          class="project-description"
+          content="project | description"
+          limit="1024"
+          use-word-boundary="true"></truncate-long-text>
+      </div>
+      <div
+        ng-if="!(project | description)">
+        <em>No description</em>
+      </div>
+    </div>
+  </div>
+</div>
+
+
+
+<form
+  name="editForm"
+  novalidate ng-if="show.editing">
+  <div row>
+    <div flex><strong>Name:</strong></div>
+    <div flex grow="4">{{projectName}}</div>
+  </div>
+  <div
+    row
+    class='form-group'
+    style="margin-top: 10px">
+    <div flex><strong>Display name: </strong></div>
```

Comment:
Should have an ID and `aria-labelledby` on the input below for accessibility. Or make it a `label` with a `for` attribute.

Here and below for description.